charts/agent-control/templates/rbac.yaml

```diff
@@ -0,0 +1,44 @@
+{{- if (index .Values "agent-control-deployment").enabled -}}
```

Comment:
same here (and in other parts of the PR)

Reply:
As commented on another thread, the problem is with dashes in the name.